events/automod.js

```diff
@@ -20,6 +23,22 @@ export async function autoMod(action) {
     const oldThread = threadChannel.threads.cache.find(
         (channel) => channel.name === `Private - ${action.member.displayName}`,
     );
+
+    let logMessage = `Member triggered auto moderation within the discord server. Reason: `;
+    if (action.matchedKeyword !== null && action.matchedKeyword !== 'game') {
```

Comment:
With all of these, could we make this a function or something to validate what kind of audit log was sent and just have the 1 set of rules vice 3? (I can do this if you need me to)